src/main/java/module/specialEvents/SpecialEventsTableModel.java

```diff
@@ -113,45 +61,29 @@ public int getRowCount() {
 
 	@Override
 	public int getColumnCount() {
-		return 15;
+		return 9;
 	}
 
 	@Override
 	public String getColumnName(int columnIndex) {
-
-		switch (columnIndex) {
-		case MATCHDATECOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Datum");
-		case MATCHIDCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("ls.match.id");
-		case HOMETACTICCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("ls.team.tactic");
-		case HOMETEAMCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Heim");
-		case AWAYTEAMCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Gast");
-		case AWAYTACTICCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("ls.team.tactic");
-		case MINUTECOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Min");
-		case SETEXTCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Event");
-		case NAMECOLUMN:
-			return HOVerwaltung.instance().getLanguageString("Spieler");
-		case CHANCECOLUMN:
-		case AWAYEVENTCOLUMN:
-		case RESULTCOLUMN:
-			return HOVerwaltung.instance().getLanguageString("ls.match.result");
-		case HOMEEVENTCOLUMN:
-		case EVENTTYPCOLUMN:
-		case MATCHTYPECOLUMN:
-			return " ";
-		default:
-			return super.getColumnName(columnIndex);
-		}
+		return switch (columnIndex) {
+			case MATCH_DATE_TYPE_COLUMN -> WordUtils.capitalizeFully(getLangStr("SpieleDetails"));
```

Comment:
I don't know if capitalising the label is a good thing, it may mess up with i18n.  The label value itself should be capitalised if this is required.

Reply:
The thing is columns were half capitalized so it was looking weired ... but I don't mind getting back to previous impl ...